infra/examples-dev/aws-all/google-workspace-variables.tf

```diff
@@ -0,0 +1,17 @@
+variable "gcp_project_id" {
+  type        = string
+  description = "string ID of GCP project that will host psoxy instance; must exist. Can leave null if not using GCP/Google Workspace."
+  default     = null
+}
+
+variable "google_workspace_example_user" {
```

Comment:
this doesn't seem to be used if google_workspace_example_admin is set.

Wouldn't be just easier to have one and say that some calls might not be available if the user hasn't certain admin rights?

Reply:
🤔 in theory, it should be the user impersonated EXCEPT for the gdirectory/google-chat/google-meet connectors, which required a privileged user (admin)